doc/users/next_whats_new/2019-05-31-AL.rst

```diff
@@ -0,0 +1,4 @@
+The *fname* argument to `FontProperties` can now be an `os.PathLike`\s
```

Comment:
What is the `\s` doing?  I don't recognize it.

Reply:
The backslash allows you to end intra-word backticks without introducing extra whitespace.